README.md

```diff
@@ -13,36 +14,142 @@ Metadata Schema and Interface Definition Language (IDL)
 $ npm install metaschema
 ```
 
-### Interface: metaschema
-
-#### create(schemas)
-
-  - `schemas: `[`<Iterable>`] schemas in form [name, schema, source] (the
-        'source' is optional)
-
-*Returns:* `[ `[`<MetaschemaError>`]`, `[`<Metaschema>`]` ]`
-
-
-Creates Metaschema instance
-
-
-#### extractDecorator(schema)
-
-  - `schema: `[`<Object>`]
-
-*Returns:* [`<string>`]
-
+## API
+
+### class Metaschema
+
+#### Metaschema.create(schemas, config)
+
+- `schemas`: `<Schema[]>`
+  - `type`: [`<string>`][string]
+  - `module`: [`<string>`][string]
+  - `name`: [`<string>`][string]
+  - `definition`: [`<Object>`][object]
+  - `source`: [`<string>`][string]
+- `config`: [`<Object>`][object]
+  - `prepare`: [`<Function>`][function]
+    - `ms`: [`<Metaschema>`][metaschema]
+  - `resolve`: [`<Function>`][function]
+    - `ms`: [`<Metaschema>`][metaschema]
+    - `type`: [`<string>`][string]
+    - `name`: [`<string>`][string]
+    - `Returns:`: `<Schema>`
+  - `processors`: [`<Object>`][object]
+    - `[type]`: [`<Object>`][object]
+      - `preprocess`: [`<Function[]>`][function] optional
+        - `schema`: `<Schema>`
+        - `Returns:`: [`<Error[]>`][error]
```

Comment:
I assume this is waiting for https://github.com/metarhia/metadoc/pull/46?

Reply:
This and also https://github.com/metarhia/globalstorage/pull/305.